code/game/objects/machinery/computer/card.dm

```diff
@@ -71,10 +71,10 @@
 		C.forceMove(src)
 		modify = C
 
-	SSnano.update_uis(src)
+	updateUsrDialog()
 	attack_hand(user)
 
-
+/*
```

Comment:
Delete

Reply:
its a todo, i'll need this code if i'm ever going to convert the id computer

Reply:
Alrighty.